api/app/Console/Kernel.php

```diff
@@ -31,7 +31,7 @@ protected function schedule(Schedule $schedule)
         $schedule->call('\App\Http\Controllers\CronController@daily')->dailyAt("17:00");
 
         // weekly cron for total week hours violation - every sunday at 10:30 pm ist
-        $schedule->call('\App\Http\Controllers\CronController@weekly')->sundays()->at('17:00');
+        $schedule->call('\App\Http\Controllers\CronController@weekly')->sundays()->at('17:30');
```

Comment:
@jalmiShreya @sharangsamonkar - can we have these values in a config instead of hardcoding.